src/react/account/details/properties/AccountKeys.jsx

```diff
@@ -0,0 +1,174 @@
+// @flow
+import { Banner, Tooltip } from '@scality/core-ui';
+import React, { useEffect, useMemo } from 'react';
+import Table, * as T from '../../../ui-elements/Table';
+import { deleteAccountAccessKey, listAccountAccessKeys } from '../../../actions';
+import { useDispatch, useSelector } from 'react-redux';
+import { useFilters, useSortBy, useTable } from 'react-table';
+import type { Account } from '../../../../types/account';
+import type { AppState } from '../../../../types/state';
+import { Clipboard } from '../../../ui-elements/Clipboard';
+import { Warning } from '../../../ui-elements/Warning';
+import { formatDate } from '../../../utils';
+import { padding } from '@scality/core-ui/dist/style/theme';
+import styled from 'styled-components';
+
+const TableContainer = styled.div`
+    display: block;
+    width: fit-content;
+    margin-top: ${padding.large};
+`;
+
+const EllipsisCell = styled.a`
+  overflow: hidden;
+  overflow-wrap: break-word;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  word-break: break-all;
+`;
+
+const initialSortBy = [
+    {
+        id: 'created_at',
+        desc: true,
+    },
+];
+
+type Props = {
+    account: Account,
+};
+
+function AccountKeys({ account }: Props) {
+    const dispatch = useDispatch();
+    const accessKeysInfo = useSelector((state: AppState) => state.user.accessKeyList);
```

Comment:
`accessKeyList` should be retrieved from the `account`  "store stable" (please create one if it does not exist) instead of `user`, to avoid confusion.